src/signals/incident-management/containers/IncidentDetail/components/MetaList/index.js

```diff
@@ -144,6 +144,20 @@ const MetaList = () => {
     [handlingTimesBySlug, incident]
   );
 
+  const [processTimeCopy, processTimeClass] = useMemo(() => {
+    const now = new Date();
+
+    if (now > new Date(incident.category?.deadline_factor_3)) {
+      return ['3x buiten de afhandeltermijn', 'alert'];
+    }
+
+    if (now > new Date(incident.category?.deadline)) {
+      return ['Buiten de afhandeltermijn', 'alert'];
+    }
+
+    return ['Binnen de afhandeltermijn', ''];
```

Comment:
Leaving out the empty string would prevent the rendered component from having an empty class (`class=""`)